apps/src/lib/levelbuilder/lesson-editor/ActivitiesEditor.story.jsx

```diff
@@ -6,14 +6,14 @@ import reducers, {
 } from '@cdo/apps/lib/levelbuilder/lesson-editor/activitiesEditorRedux';
 import {Provider} from 'react-redux';
 import {
-  levelKeyList,
-  activities
-} from '@cdo/apps/lib/levelbuilder/lesson-editor/SampleActivitiesData';
+  sampleActivities,
+  levelKeyList
+} from '../../../../test/unit/lib/levelbuilder/lesson-editor/activitiesTestData';
```

Comment:
Are we importing something defined in a test file into shipping code here?  This seems unexpected?

Reply:
This is a storybook component so its more of a test than shipped code.

Reply:
in this case the answer is "no" because .story.jsx are built on [storybook](https://storybook.js.org/) and are not shipping code. We use storybook to be able to display individual react components outside of any shipping feature. this speeds up development because storybook components reload super quickly. before Dani had a whole working page, she was able to build individual react components and see what they would look like using fake data and storybook.

Storybook can potentially be used to make a library of sorts of all the UI components in our system so that product managers can select from it when designing new features, but we haven't really been using it that way.

Reply:
Oh, interesting.  Thanks for the explanation!